ui/app/serializers/job.js

```diff
@@ -71,7 +71,13 @@ export default class JobSerializer extends ApplicationSerializer {
     return super.normalize(typeHash, hash);
   }
 
-  normalizeQueryResponse(store, primaryModelClass, payload, id, requestType) {
+  normalizeQueryResponse(
+    store,
+    primaryModelClass,
+    payload = [],
```

Comment:
Only notable change here is a default of payload to `[]` so I don't have to do a bunch of conditional guards with [elvis operators](https://en.wikipedia.org/wiki/Elvis_operator) later